rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -81,6 +81,7 @@ void SequentialWriter::open(
 {
   max_bagfile_size_ = storage_options.max_bagfile_size;
   base_folder_ = storage_options.uri;
+  chunk_size_ = storage_options.chunk_size;
```

Comment:
the chunk size can also be used to `reserve` space in the `cache_` vector so a call to `push_back` doesn't reshuffle the storage.